packages/blade/scripts/generateBundleSizeInfo.js

```diff
@@ -0,0 +1,95 @@
+const fs = require('fs');
+const path = require('path');
+const babelParser = require('@babel/parser');
+const traverse = require('@babel/traverse').default;
+const execa = require('execa');
+
+const main = () => {
+  // Get all the components name exported from the bundle and add them to the size-limit configuration
+  const fileContent = fs.readFileSync(
+    path.resolve(__dirname, '../build/lib/web/production/components/index.js'),
+    'utf8',
+  );
+
+  // Parse the file content to generate an AST
+  const ast = babelParser.parse(fileContent, {
+    sourceType: 'module',
+  });
+
+  // Arrays to store size-limit stats and component names to exclude
+  const sizes = [];
+  const excludedComponents = [
+    'useTheme',
+    'BladeProvider',
+    'announce',
+    'clearAnnouncer',
+    'destroyAnnouncer',
+    'getTextProps',
+    'screenReaderStyles',
+    'useActionListContext',
+    'ActionListItemAsset', // This is just an <img> tag
+    // These are same as Badge, Amount, Counter, Link, Text components
+    'CardHeaderAmount',
+    'CardHeaderBadge',
+    'CardHeaderCounter',
+    'CardHeaderLink',
+    'CardHeaderText',
```

Comment:
Any harm in keeping these? They are wrappers but their size might increase decrease depending on the code we add later no.

Reply:
It just increases the script run time, these components can add approx ~2mins. I believe these wrappers would hardly get updates. They have dev-only checks to verify amount is being used inside card only.
https://github.com/razorpay/blade/blob/8d89eb5365f75880ac3c07ce054d8633c3f05422/packages/blade/src/components/Card/CardHeader.tsx#L57-L61